CHANGES.rst

```diff
@@ -2,6 +2,9 @@
 Changelog
 =========
 
+* Validate YAML indicators description before trying to build module. (:issue:`1523`)
```

Comment:
```suggestion
v0.48.0 (unreleased)
--------------------
Contributors to this version: David Huard (:user:`huard`),

New features and enhancements
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
* Validate YAML indicators description before trying to build module. (:issue:`1523`)
```

Reply:
Yeah, I knew I'd have to fiddle there after the merge of the uncertainty partitioning method, so didn't bother.